src/components/organisms/about/about.scss

```
@@ -0,0 +1,26 @@
+@use "../../variables" as *;
+.about {
+  margin-top:100px;
+  margin-bottom:100px;

```

marginは相殺のリスクがあるので、パーツの上下方向につけるのはあまりお勧めしません。
今回のように余白を取りたい場合は padding を使っていきましょう。

```suggestion
  padding: 100px 0;
```

参考：https://coliss.com/articles/build-websites/operation/css/about-collapsing-margins.html